tests/test_save.py

```diff
@@ -131,3 +140,30 @@ def test_model_manipulation(request, model_class, storage_method):
     finally:
         if os.path.exists(model_fname):
             os.remove(model_fname)
+
+@pytest.mark.parametrize("model_class", MODEL_LIST)
+def test_save_custom_objects(request, model_class):
+    """
+    Test feeding custom_objects in model.load(...) function
+    """
+    model_fname = './test_model_{}.model'.format(request.node.name)
+
+    try:
+        env = DummyVecEnv([lambda: IdentityEnv(10)])
+        model = model_class(policy="MlpPolicy", env=env)
+        model.save(model_fname)
+
+        del model, env
+
+        model = model_class.load(
+            model_fname,
+            custom_objects={"n_envs": 10}
```

Comment:
I would also try with a custom policy because in practice, `custom_objects` will be useful for objects that serialized using pickle, no?

Reply:
Will implement, just need to think carefully how to do this sensibly.